server_tests/test/clientTests.js

```diff
@@ -1422,7 +1413,8 @@ describe("RUST API TEST SUITE", function() {
                 if (timeElapsed > maxAcceptableTime) {
                     assert.fail('operation took too long. timeElapsed : ' + timeElapsed + ' > maxAcceptableTime : ' + maxAcceptableTime);
                 } else {
-                    Comparison.bublTimelapseOutput(res, expectedResults);
+                    validate.done(res.body, schema.names.commandBublTimelapse);
+                    assert.notEqual(res.body.results.fileUri, expectedResults);
```

Comment:
This is not correct. What we're trying to assert here is that `res.body.results._bublFileUris.length` is equal to `timelapseCount`.